docs/zh/sql-reference/sql-statements/table_bucket_part_index/SELECT.md

```diff
@@ -1136,3 +1136,56 @@ SELECT SUM(CASE WHEN c3 = 1 THEN c1 ELSE NULL END) AS sum_c1_1,
 FROM t1
 GROUP BY c0;
 ```
+
+### EXCLUDE
+
+该函数从 4.0 版本开始支持。
```

Comment:
why version 4.0?

Reply:
https://github.com/StarRocks/starrocks/pull/57411#discussion_r2028964910


The original version was 3.6, but it was later changed to 4.0 based on @jaogoy 's comments.